pull_request_template

```diff
@@ -0,0 +1,42 @@
+## Status
+**READY/IN DEVELOPMENT/HOLD**
+
+## Backport
+YES | NO
+
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+## Related PRs
```

Comment:
Let's change that to "Related Issue(s)" - we've already decided during our 1st retrospective that each PR must be associated with an issue.